abstract-account-keys/src/lib.rs

```diff
@@ -0,0 +1,31 @@
+#![no_std]
+#![deny(missing_docs)]
+
+//! This crate provides traits that either AccountKeys objects or Hardware
+//! Wallets objects can implement to support signing a Tx, checking balance,
+//! etc.
+
+// FIXME
```

Comment:
What's this FIXME for?

Reply:
```suggestion
// FIXME: stop depending on alloc. This crate needs to be core-only to work on hardware wallets
```

Reply:
i think the abstract parts of this could end up in `mc_core` now?